### ISSUE TITLE
Firefox: New tab (control +T) break the scroll 

### ISSUE THREAD

**Issue by CricriSDN** [alvarotrigo/fullPage.js#1472]
Hi,
Thanks first for this beautiful plugin and apologize for my bad english :) I don't know if this issue is already here but when you use (i use it a lot) the shortcut new Tab on FF (ctrl + T), the tab open, and if you come back to the fullPage website, you loose the scroll... It works with Chrome but not in IE too (win8).

First time, I imagine it comes from the website (use foundation, velocity) and think that there was a problem with one jquery plugin. But issue is still on the fullPage.js demo.

Before trying to delete all the plugins on FF, is there someone here who has the same issue ?

**Comment by alvarotrigo** [comment]
Use the latest version. (2.6.9) The problem was fixed in 2.6.6 as you can see in the [change log](https://github.com/alvarotrigo/fullPage.js/releases)

> Fixed bug when using scrollBar:true and opening a new tab (ctrl + t)

**Comment by CricriSDN** [comment]
Thanks for your answer Alvaro. But I alreday use 2.6.9, scrollBar is false (by default i think) and i meet this issue with Live demo : http://alvarotrigo.com/fullPage/

1. load the demo
2. press control + T
3. come back to the live demo, scroll is broken

**Comment by alvarotrigo** [comment]
Oh right, you are speaking about Firefox. I'll change the title and reopen it.

**Comment by CricriSDN** [comment]
Yes, and issue is the same with IE10 (win8). It sems to work well with Chrome, may be the blur function ? :/

**Comment by CricriSDN** [comment]
The if statement on line 1080 is working on Chrome, not with FF and IE
`if(options.autoScrolling && !controlPressed){`

*autoScrolling* is not set and it is true by default, so I check *controlPressed* but it looks fine...

**Comment by alvarotrigo** [comment]
Fixed in fullpage.js 2.7.0

**Comment by CricriSDN** [comment]
Thank you Alvaro, works with FF and IE.

But now i have
* conflict with zurb foundation (reveal modal break the js)
* section height is not defined well - have a border on the bottom (work fine with fullpage 2.6.6)
* no scroll in Google Chrome, the section swap directly to another

Great todo for this evening ! :)

**Comment by alvarotrigo** [comment]
@CricriSDN could you provide a reproduction of those problems? 



> section height is not defined well - have a border on the bottom (work fine with fullpage 2.6.6)

>no scroll in Google Chrome, the section swap directly to another

[The examples online](http://alvarotrigo.com/fullPage/) work ok.

**Comment by CricriSDN** [comment]
Thanks Alvaro for your answer. Unfortunately, the project is on local dev server. I will try to investigate before...

**Comment by CricriSDN** [comment]
2.7.1 resolve conflict with zurb modal

**Comment by CricriSDN** [comment]
Just resolve section height (some of my items with absolute position and transform origin...).

For the issue with scroll on Chrome, may be you have an idea ? If i disable css3 (change to false), scroll works. If true, there is no scroll (only after afterRender). Because i use callback afterLoad (animation), i can see that scrollingspeed param is ok, if it's longer, the time to have animation is longer.

**Comment by alvarotrigo** [comment]
> For the issue with scroll on Chrome, may be you have an idea ?

I don't know what issue are you talking about I'm afraid. Try to reproduce it isolating the problem in jsfiddle or codepen.

**Comment by CricriSDN** [comment]
Big Thanks for your answer Alvaro. Currently we manage the issue with Chrome by changing css3 to false. I will try to investigate further and open new post for this issue if i can't find the problem.

**Comment by CricriSDN** [comment]
Hi Alvaro
Still have the problem with height size after afterRender. Issue in on the demo page too, so I open new issue to make it easier to follow :)
